ansible/vars.yml

```diff
@@ -0,0 +1,21 @@
+workspace: /var/www/AcmeProject
+
+timezone: Europe/Paris
+
+sites: sites
+
+mysql:
+  databases:
+    - { name: 'test' }
+  users:
+    - { name: 'test' }
+
+rabbitmq:
+  users:
+    - { name: 'admin' }
+
+php_version: '5.6'
+
+etc_hosts_lines:
+  - { ip: '192.168.100.12', hostname: 'my-other-vm-contains-redis-and-elasticsearch' }
+  - { ip: '192.168.100.12', hostname: 'microservice-hosted-elsewhere.dev' }
```

Comment:
these /etc/hosts lines won't be useful at all, I think you shouldn't set this variable at all, because we have no idea of what will be needed in the user project